habitat_baselines/rl/ppo/utils.py

```diff
@@ -422,4 +425,72 @@ def ppo_args():
         nargs=argparse.REMAINDER,
         help="Modify config options from command line",
     )
+    parser.add_argument(
```

Comment:
Add an option to not specify the `--tensorboard-dir`, in case someone does not want to use tensorboard.